content/pages/disability-benefits/claims-appeal/hearings.md

```diff
@@ -6,63 +6,126 @@ plainlanguage: 11-4-16 certified in compliance with the Plain Writing Act
+  -	**An in-person hearing at the Board in Washington, DC.**
+
+[Download VA Form 9](https://www.va.gov/vaforms/va/pdf/va9.pdf).
+
+Or, you can contact your Veterans Service Organization to get help with requesting a hearing at any time during the appeals process.<br>
+[Find a Veterans Service Organization](https://www.va.gov/vso/).
+
+### How will I know when my hearing is scheduled?
+
+You’ll receive a notice in the mail at least 30 days before your hearing is scheduled.
+
+### What if I need to reschedule my hearing?
+
+Send a written request to reschedule your hearing at least 2 weeks before your scheduled hearing. Include your name and the VA file number for your claim.
+
+**Mail your request to:**
+
 <p class="va-address-block">
-  Director, Office of Management, Planning and Analysis (014)<br/>
-  Board of Veterans’ Appeals<br/>
-  PO Box 27063<br/>
-  Washington, DC 20038<br/>
+Board of Veterans’ Appeals<br>
+PO Box 27063<br>
+Washington, DC 20038<br>
 </p>
-<p>If you’re able to reschedule your hearing, you’ll receive a notice at least 30 days before the new hearing date.</p>
 
-<h3>What if I have to cancel my hearing?</h3>
-<p>If you need to cancel your hearing, please send a letter to the Board at least 2 weeks before your hearing, so the VLJ can hear another appeal at that time. The VLJ will make a decision based on the information you’ve provided in your appeal.</p>
-<p>Send the letter to:</p><br/>
+**You'll need to file a motion explaining why you have "good cause" for rescheduling any of these types of hearings:
+
+- An in-person hearing at the Board in Washington, DC that's less than 2 weeks away, **or**
+- An in-person hearing at the Board in Washington, DC if you already rescheduled your hearing before, **or**
+- An in-person hearing at your local VA regional office (called a Travel Board hearing), **or**
+- A video hearing
+
+Examples of "good cause" might be if you, your representative, or a witness are sick, or if you have trouble getting the records you need to support your appeal.
+
+A judge will review your request and decide if the hearing can be rescheduled. You’ll get a copy of the decision about rescheduling your hearing in the mail. If VA is able to reschedule your hearing, you’ll receive a notice at least 30 days before the new hearing date.
+
+### What if I need to cancel my hearing?
+
+Send a written request to cancel your hearing at least 2 weeks before your scheduled hearing. Include your name, the VA file number for your claim, and the reason you're cancelling.
+
+**Mail your request to:**
+
 <p class="va-address-block">
-  Director, Office of Management, Planning and Analysis (014)<br/>
-  Board of Veterans’ Appeals<br/>
-  PO Box 27063<br/>
-  Washington, DC 20038<br/>
+Board of Veterans’ Appeals<br>
+PO Box 27063<br>
+Washington, DC 20038<br>
 </p>
+
+<span id="preparing"></span>
```

Comment:
There are a few of these blank tags. What are they for?

Reply:
These are skip links. We have set them up this way throughout, however, we should probably use [sections](https://webaim.org/techniques/skipnav/) instead. Ticket created [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9186).

Reply:
Ahhh, skip links. Makes sense. Thanks! :+1: